docs/cromwell_features/CallCaching.md

```diff
@@ -47,23 +41,83 @@ call-caching {
   invalidate-bad-cache-results = false
 
   blacklist-cache {
-    # The call caching blacklist cache is off by default. This is used to blacklist cache hit paths based on the
-    # prefixes of cache hit paths that Cromwell previously failed to copy for authorization reasons.
-    enabled: true
-    # Guava cache concurrency.
-    concurrency: 10000
-    # How long entries in the cache should live from the time of their last access.
-    ttl: 20 minutes
-    # Maximum number of entries in the cache.
-    size: 1000
+     # The call caching blacklist cache is off by default. This cache is used to blacklist cache hits based on cache
+     # hit ids or buckets of cache hit paths that Cromwell has previously failed to copy for permissions reasons.
+     enabled: true
+
+     # All blacklisting values below are optional except for groupings.workflow-option.
```

Comment:
I thought it was optional, and if it's not specified then Cromwell acts like it's every workflow for itself?

Reply:
Yes that's correct, I guess what I wrote is unclear. What I was trying to say is that if one wants to use the blacklist cache grouping support then `workflow-option` needs to be defined, otherwise the blacklist cache would be scoped to the root workflow. Will fix.